BraintreeCore/src/test/java/com/braintreepayments/api/core/AnalyticsClientUnitTest.kt

```diff
@@ -103,58 +123,63 @@ class AnalyticsClientUnitTest {
         assertEquals(authorization.toString(), workSpec.input.getString("authorization"))
         assertEquals("sample-session-id", workSpec.input.getString("sessionId"))
         assertEquals("sample-integration", workSpec.input.getString("integration"))
-    }
 
-    @Test
-    fun writeAnalytics_whenEventNameAndTimestampArePresent_returnsSuccess() {
-        val inputData = Data.Builder()
-            .putString(AnalyticsClient.WORK_INPUT_KEY_EVENT_NAME, eventName)
-            .putLong(AnalyticsClient.WORK_INPUT_KEY_TIMESTAMP, timestamp)
-            .build()
-        val sut = AnalyticsClient(httpClient, analyticsDatabase, workManager, deviceInspector)
-        val result = sut.writeAnalytics(inputData)
-        assertTrue(result is ListenableWorker.Result.Success)
+        // TODO: Fix this test
```

Comment:
q: Is this a leftover note from the migration? We can remove if the test is passing. It looks identical to the test we have in `main`.

Reply:
whoops! i forgot to remove this. great catch!